zingolib/src/config.rs

```diff
@@ -726,3 +730,42 @@ impl ActivationHeights {
         }
     }
 }
+
+mod tests {
+
+    #[tokio::test]
+    async fn test_load_clientconfig_serverless() {
+        rustls::crypto::ring::default_provider()
+            .install_default()
+            .expect("Ring to work as a default");
+        tracing_subscriber::fmt().init();
+
+        let valid_uri = crate::config::construct_lightwalletd_uri(Some(
+            crate::config::DEFAULT_LIGHTWALLETD_SERVER.to_string(),
+        ));
+        // let invalid_uri = construct_lightwalletd_uri(Some("Invalid URI".to_string()));
+        let temp_dir = tempfile::TempDir::new().unwrap();
+
+        let temp_path = temp_dir.path().to_path_buf();
+        // let temp_path_invalid = temp_dir.path().to_path_buf();
+
+        let valid_config = crate::config::load_clientconfig(
+            valid_uri.clone(),
+            Some(temp_path),
+            crate::config::ChainType::Mainnet,
+            true,
+        )
+        .unwrap();
+
+        assert_eq!(valid_config.get_lightwalletd_uri(), valid_uri);
```

Comment:
this make no sense to me... in a serverless scenario... in fact the normal situation for `serverless` is:
- server uri: empty.
- chain type: empty.